### ISSUE TITLE
Add ‘Collaborators’ section 

### ISSUE THREAD

**Issue by Aliserag** [Aliserag/CovidImpact#3]
**Overview:**
To highlight all the coding ninjas, professors and small businesses that contributed to the project to recognize their work and involvement.

**What needs to be done:**
Steps

1. Create a list of everyone who contributed from the repo as well as asking @Aliserag about the non-tech contributors
2. Create a new page called Contributors
3. Each contributor has a title, a name and an avatar icon linking to more information about them (either their github profile or linkedin or personal website.  Ensure all images follow the same style.
4. Add text that links to the new page at the bottom of the homepage 'Click _**here**_ to meet the team! Or add a team link in the top-bar.

**Criteria of success:**
Ability to see all the contributors in a fun format by navigating from the homepage.

**Documentation:**
Can get icons from undraw.co or https://www.flaticon.com/ (really anywhere). An example can be:

**_Code Ninja_**
![ninja](https://user-images.githubusercontent.com/11467748/80147918-1777aa00-8569-11ea-9ce5-2c8846b14298.png)
Bolat Khokayev

**Stakeholder:**
@Aliserag

**Comment by NoirKomBatman** [comment]
@Aliserag @aPruner @albertocevallos  does anyone know a way to auto-parse the committers from GitHub and use that info in the collaborators section? Is it a good idea?

**Comment by aPruner** [comment]
@NoirKomBatman Yup! We can likely use github webhooks for this: https://developer.github.com/webhooks/ I haven't used them before but I'm sure it's not too difficult!
